fragments/k8s/core/ci-info.yaml

```diff
@@ -0,0 +1,18 @@
+# This is an incomplete ci-info.yaml fragment.
+bundle:
+  namespace: kos.tsakalozos
```

Comment:
No this is not correct. If we decide to merge this PR this will change to container. As you mention I need this here for now for testing.